.github/workflows/ihsBuild.yml

```diff
@@ -195,11 +218,30 @@ jobs:
           dataDiskSasUrl=${blobStorageEndpoint}vhds/${{ env.vmName }}datadisk1.vhd?$sasToken
           
           echo "osDiskSasUrl: ${osDiskSasUrl}, dataDiskSasUrl: ${dataDiskSasUrl}" > sas-url-ihs.txt
+          echo "##[set-output name=osDiskSasUrl;]${osDiskSasUrl}"
+          echo "##[set-output name=dataDiskSasUrl;]${dataDiskSasUrl}"
       - name: Upload SAS url
         uses: actions/upload-artifact@v3
         with:
           name: sasurl-ihs
           path: sas-url-ihs.txt
+      - name: Update offer sas url and version
+        uses: microsoft/microsoft-partner-center-github-action@v3
+        if: ${{ github.repository_owner == 'WASdev' }}
+        with:
+          clientId: ${{ env.clientId }}
+          dataDiskSasUrl: ${{steps.sas_url.outputs.dataDiskSasUrl}}
+          imageType: ${{ env.imageType }}
+          imageVersionNumber: ${{ steps.setup-env-variables-based-on-dispatch-event.outputs.imageVersionNumber }}
+          offerId: ${{ env.offerId }}
+          offerType: ${{ env.offerType }}
+          operatingSystemFamily: ${{ env.operatingSystemFamily }}
+          operatingSystemType: ${{ env.operatingSystemType }}
+          osDiskSasUrl: ${{steps.sas_url.outputs.osDiskSasUrl}}
+          planId: ${{ env.planId }}
+          secretValue: ${{ env.secretValue }}
+          tenantId: ${{ env.tenantId }}
+          verbose: "true"
```

Comment:
Personally, I prefer group variables per category instead of alphabetical order, which seems easier to check if anyone is missed. For example:
```
          offerId: ${{ env.offerId }}
          planId: ${{ env.planId }}
          offerType: ${{ env.offerType }}
          imageVersionNumber: ${{ steps.setup-env-variables-based-on-dispatch-event.outputs.imageVersionNumber }}
          imageType: ${{ env.imageType }}
          operatingSystemFamily: ${{ env.operatingSystemFamily }}
          operatingSystemType: ${{ env.operatingSystemType }}
          osDiskSasUrl: ${{steps.sas_url.outputs.osDiskSasUrl}}
          dataDiskSasUrl: ${{steps.sas_url.outputs.dataDiskSasUrl}}
          clientId: ${{ env.clientId }}
          secretValue: ${{ env.secretValue }}
          tenantId: ${{ env.tenantId }}
          verbose: "true"
```